### PR TITLE
rpc: Always return sequencer address in BlockHeader

### LINKED ISSUE #1117: Make sure we always return a sequencer address in block header
RPC spec makes the sequencer address required.
Let's just return 0 if core.Header.SequencerAddress is `nil`

### PR DESCRIPTION
**Pull request by Exca-DK**
closes #1117

### CODE REVIEW

File: rpc/handlers.go [hunk 1]
```diff
@@ -174,13 +174,18 @@ func adaptBlockHeader(header *core.Header) BlockHeader {
 		blockNumber = &header.Number
 	}
 
+	sequencerAddress := header.SequencerAddress
+	if sequencerAddress == nil {
+		sequencerAddress = new(felt.Felt)
```
Comment (omerfirmak) [inline rpc/handlers.go:179]:
```suggestion
		sequencerAddress = &felt.Zero
```

**Review by omerfirmak — COMMENTED** [review]
Thanks! Do you mind adding a test as well?

**Comment by Exca-DK** [comment]
> Thanks! Do you mind adding a test as well?

What kind of test? Checking returned header or comparing JSON dump?

**Review by omerfirmak — APPROVED** [review]
Perfect, thank you!

**Pull request merged**